projects/canopy/src/lib/modal/modal-body-timer/modal-body-timer.component.ts

```diff
@@ -13,8 +14,16 @@ import {
   encapsulation: ViewEncapsulation.None,
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class LgModalBodyTimerComponent {
-  @Input() timer: string;
+export class LgModalBodyTimerComponent implements OnInit {
+  @Input() timer: number;
+  formattedTime: string | number;
```

Comment:
can the return type here be just a string? I'm trying to work out why you've got a union type here

Reply:
It can and should be 👍🏻 updating